packages/neuron-ui/src/utils/enums.ts

```diff
@@ -141,27 +141,47 @@ export enum DefaultLockInfo {
   HashType = 'type',
   CodeHashIndex = '0x00',
   ArgsLen = '20',
+  TagName = 'secp256k1_160',
```

Comment:
It's neither `secp256k1/blake160` in [RFC](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0024-ckb-genesis-script-list/0024-ckb-genesis-script-list.md#secp256k1blake160) nor `secp256k1_blake160` used in ckb explorer https://explorer.nervos.org/address/ckb1qyqqcnfvwdvd2zmhf0zy2csmldf8dzz5qadszd3nf6

Reply:
I've found the tags in ckb explorer is not exact, say `secp256k1 / multisig / locktime` in https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/constants/scripts.ts#L21

So we can just follow the RFC, it's 100% official